Dockerfile.armv7

```diff
@@ -1,5 +1,10 @@
-FROM arm32v7/alpine:3.12
-MAINTAINER Marc Crébassa <aalaesar@gmail.com>
+ARG QEMU_IMAGE
+FROM ${QEMU_IMAGE} as qemu
+
+FROM arm32v7/alpine:3.12 as base
+COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin/
+
+MAINTAINER LoZ Open Source Ecosystem (https://www.ibm.com/developerworks/community/groups/community/lozopensource)
```

Comment:
Why this change?  Probably better to have a separate PR for this, so that the reason for the change is clearly described.

Reply:
Since `arm32v7/alpine` is an `arm` based image and in it executable files are `arm` only we neeed the qemu binary to run the commands like the following one
https://github.com/projectcalico/calicoctl/blob/136af0e96783676c7faba39c71b1ebf390d04dd6/Dockerfile.armv7#L9